docker-compose/_posts/2017-05-05-adding-a-database-for-compose.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Adding a Database for Compose
+short: Adding a Database
+category: docker-compose
+order: 2
+---
+
+Learn how to add a database to your Compose based environment.
+
+> If you’re **not using Compose**, follow the [databases for Dockerfiles guide](https://runnable.com/docs/databases-datastores/how-to-setup-your-postgres-database-template).
+
+---
+
+Start by creating a Dockerfile to build your service and adding it to your repository:
+
+<div class="grid-block align-center pre-header">
```

Comment:
Yeah, i figured it was better to show both instead of only one if JS was off. What do you think?